int_time: 400.0
wait_time: 600.0
bias: 1
vth_value: 256
timeout: 1000
